modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -605,6 +605,9 @@ public class IgniteConfiguration {
     /** Shutdown policy for cluster. */
     public ShutdownPolicy shutdown = DFLT_SHUTDOWN_POLICY;
 
+    /** Default values for distributed properties. */
+    private Map<String, ?> distrProps;
```

Comment:
Let's have Map<String, String> here as we support setting properties from String and this will simplify API.